c2corg_ui/static/js/viewdetails.js

```diff
@@ -133,6 +142,27 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: this.compile_(template)(this.scope_)});
 };
 
+/**
+ * @param {Object} doc
+ * @private
+ */
+app.ViewDetailsController.prototype.removeAssociation_ = function(doc) {
+  var associations;
+
+  if (doc['type'] === 'outings') {
+    associations = this.scope_['document']['associations']['recent_outings']['outings'];
+  } else {
+    associations = this.scope_['document']['associations'][doc['type']];
+  }
+  if (associations) {
+    for (var i = 0; i < associations.length; i++) {
```

Comment:
Do you want to refactor this in a function?
https://github.com/c2corg/v6_ui/pull/328/files#diff-8f682851013ffc3282976f742de82930R140